dotcom-rendering/src/components/FlexibleSpecial.tsx

```diff
@@ -31,6 +32,19 @@ export const OneCardLayout = ({
 }) => {
 	if (!cards[0]) return null;
 
+	function getCardImageSize(boostLevel: string | undefined): ImageSizeType {
```

Comment:
This bit is honestly optional, but does serve as a nice small proof of concept applying the new property.

Reply:
This is interesting. My understanding was that the image size wasn't actually affected by the boost level as it is always either 75% or 100%. Do you have examples where we want a image size other than jumbo?

Reply:
Maybe proof of concept was the wrong term, mainly just wanted to have an easily visible, placeholder kind of change to show different levels making an impact. Hadn't really considered how the image gets sized, but it does look like jumbo works overall. Happy to remove this though!